packages/desktop-client/src/components/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+import * as Platform from 'loot-core/src/client/platform';
+
+import Reports from '../../icons/v1/Reports';
+import Wallet from '../../icons/v1/Wallet';
+import CalendarIcon from '../../icons/v2/Calendar';
+// eslint-disable-next-line no-restricted-imports
+import { colors } from '../../style';
+import View from '../common/View';
+
+import { Accounts } from './Accounts';
+import { Item } from './Item';
+import { ToggleButton } from './ToggleButton';
+import { Tools } from './Tools';
+
+import { useSidebar } from '.';
+
+export const SIDEBAR_WIDTH = 240;
+
+export function Sidebar({
+  style,
+  budgetName,
+  accounts,
+  failedAccounts,
+  updatedAccounts,
+  getBalanceQuery,
+  getAllAccountBalance,
+  getOnBudgetBalance,
+  getOffBudgetBalance,
+  showClosedAccounts,
+  isFloating,
+  onFloat,
+  onAddAccount,
+  onToggleClosedAccounts,
+  onReorder,
+}) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏missing prop type
```suggestion
}: SidebarProps) {
```